src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -478,11 +478,31 @@ private void updateChunkLoadTimer()
                             ((ServerChunkProvider) world.getChunkProvider()).releaseTicket(KEEP_LOADED_TYPE, pos, 2, pos);
                         }
                     }
+                    ticketedChunks.clear();
+                    ticketedChunksDirty = true;
                 }
             }
         }
     }
 
+    /**
+     * Checks the chunk and registers a ticket for it if needed
+     *
+     * @param chunkPos chunk position to check
+     */
+    private void checkChunkAndRegisterTicket(final long chunkPos, final Chunk chunk)
```

Comment:
Doesn't this need the permission check too?

Reply:
not really, since the timer includes that